src/main/java/org/dependencytrack/persistence/FindingsSearchQueryManager.java

```diff
@@ -128,6 +129,10 @@ public PaginatedResult getAllFindings(final Map<String, String> filters, final b
             final Finding finding = new Finding(UUID.fromString((String) o[29]), o);
             final Component component = getObjectByUuid(Component.class, (String) finding.getComponent().get("uuid"));
             final Vulnerability vulnerability = getObjectByUuid(Vulnerability.class, (String) finding.getVulnerability().get("uuid"));
+            final Project project = component.getProject();
+            if (project != null) {
+                finding.getComponent().put("tags", project.getTags());
+            }
             final Analysis analysis = getAnalysis(component, vulnerability);
```

Comment:
Should it be made more clear that these are **project** tags and not finding tags? A future DT version might have finding level tags. Name the field `project-tags`?

Reply:
@valentijnscholten Totally agree. Will make the change!